tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -63,15 +63,15 @@ class REntry;
 
 namespace Internal {
 struct RFieldCallbackInjector;
+class RPageStorage;
 // TODO(jblomer): find a better way to not have these three methods in the RFieldBase public API
 void CallCommitClusterOnField(RFieldBase &);
-void CallConnectPageSinkOnField(RFieldBase &, Detail::RPageSink &, NTupleSize_t firstEntry = 0);
-void CallConnectPageSourceOnField(RFieldBase &, Detail::RPageSource &);
+void CallConnectPageSinkOnField(RFieldBase &, Internal::RPageSink &, NTupleSize_t firstEntry = 0);
+void CallConnectPageSourceOnField(RFieldBase &, Internal::RPageSource &);
```

Comment:
I assume we should also forward-declare `RPageSink` and `RPageSource` (I guess at the moment that's done by one of the included headers for us)